source/style/content.css

```
@@ -221,6 +221,27 @@ code.refined-twitter_markdown {
 	max-height: initial !important;
 }
 
+/* Alternative text for user avatars */
+.twitter_image-alt_profile-container {
```

You add `refined-twitter_image-alt_profile-container` but these don't have the `refined-` prefix :)
```suggestion
.refined-twitter_image-alt_profile-container {
```

---

Aw snap. You're absolutely right, I fixed that locally in the `/distribution` instead of the `/source`. 🙈